app-multimedia/deadbeef/autobuild/defines

```diff
@@ -10,7 +10,12 @@ AUTOTOOLS_AFTER="--enable-gtk3 \
                  --disable-gtk2 \
                  --enable-ffmpeg \
                  --enable-cdda \
-                 --enable-vfs-zip"
+                 --enable-vfs-zip \
+                 --disable-libretro"
+
+# SSE3 is required for libretro plugin
+AUTOTOOLS_AFTER__AMD64="${AUTOTOOLS_AFTER} --enable-libretro"
```

Comment:
Uh, in that case I would just disable `libretro` unless we have a way to limit it to SSE2, which is our current baseline.